cql3/functions/user_function.cc

```diff
@@ -42,6 +42,9 @@ bytes_opt user_function::execute(cql_serialization_format sf, const std::vector<
     if (!seastar::thread::running_in_thread()) {
         on_internal_error(log, "User function cannot be executed in this context");
     }
+    // Possibly yield between each function execution, to avoid reactor stalls
+    // when aggregating a large input.
+    seastar::thread::maybe_yield();
```

Comment:
I thought lua execution already contains yields. So the wasm fix should be sufficient.

Reply:
lua indeed yields when there's a need to preempt and then proceeds to sleep on the next repeat_until_value() iteration. I am withdrawing this PR then...